s3/replication/replicator/src/s3replicator/object_tag_replicator.py

```diff
@@ -97,6 +98,39 @@ async def start(self):
             else:
                 await observer.notify(JobEvents.COMPLETED, self._job_id)
 
+        if JobEvents.COMPLETED:
+            # check object tags count of source and target objects
+            # [user-defined metadata]
+            target_response = None
+            command = 'aws s3api get-object --bucket ' + \
+                self._target_bucket + ' --key ' + self._target_object + ' outfile.txt >> tagset.py'
```

Comment:
doesn't need to be a python file. it can be a normal file

Reply:
yes. done.